### DISCUSSION TITLE
ViT-G/14 training hyperparameter

### DISCUSSION THREAD

**Discussion by howardhsu** [mlfoundations/open_clip#628] [Q&A]
After reading this post (https://laion.ai/blog/giant-openclip/), do we have the exact training hyperparameters of ViT-G/14 somewhere?  I cannot find it on rom1504's wandb.

**Comment by mitchellnw** [comment]
- LR 2e-3
- AdamW beta2 0.95
- Warmup 10k
- Batch size 160k
- Patch drop 0.5

Any others you're interested in?

**Comment by howardhsu** [comment]
Thx for sharing. How many steps/seen pairs for phase 1 and phase 2? Did phase 2 resume optimizer states from phase 1?

**Comment by mitchellnw** [comment]
I think this is the blog post but 32B phase 1 and ~7B phase 2 I believe. And yes, re: opt states.

**Comment by howardhsu** [comment]
Ok, sounds clear to me, thx for quick reply. will impl. one. What's the steps of gradient accumulation used or just make sure seen pairs per update similar as phase1?

**Comment by mitchellnw** [comment]
Sorry I don't understand

**Comment by howardhsu** [comment]
Sorry for confusion. The blog post https://laion.ai/blog/giant-openclip/   mentioned gradient accumulation used in phase 2, not sure the steps to accumulate per weight update.

**Comment by mitchellnw** [comment]
Oh, 2

**Comment by rwightman** [comment]
moving to discussions for future reference...

**Comment by leng-yue** [comment]
Will using gradient accumulation hurt the accuracy if I am only using 256 GPUs (comparing to 800 or 1600)?
